### PR TITLE
HPCC-15520 Fix usage issues

### PR DESCRIPTION
**Pull request by rpastrana**
Signed-off-by: rpastrana rodrigo.pastrana@lexisnexis.com

### CODE REVIEW

**Comment by rpastrana** [comment]
@wangkx please review

File: tools/esdlcmd/esdl-publish.cpp [hunk 1]
```diff
@@ -75,10 +75,10 @@ class EsdlPublishCmdCommon : public EsdlCmdCommon
     virtual void usage()
     {
          printf(
-                "   -s, --server <ip>            IP of server running ESDL services\n"
+                "   -s, --server <ip>            IP of server running WsESDLConfig service\n"
                 "   --port <port>                ESDL services port\n"
```
Comment (wangkx) [inline tools/esdlcmd/esdl-publish.cpp]: WsESDLConfig service port?

**Comment by wangkx** [comment]
@rpastrana one question only.

**Comment by rpastrana** [comment]
@wangkx please review.

**Comment by wangkx** [comment]
@rpastrana looks fine.

**Comment by rpastrana** [comment]
@richardkchapman please merge

**Pull request merged**